### PR TITLE
Feature/graph 306

### PR DESCRIPTION
**Pull request by Seb-figment**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
+.PHONY: generate format
 generate:
 	@mkdir -p ./codec
 	@protoc \
-		--proto_path ./ \
 		--go_out=paths=source_relative:./codec \
-		*.proto
+		codec.proto
+format:
+	@clang-format -i --style=file *.proto
```
Comment (lavabyrd) [inline Makefile:8]: new line :)

**Comment by lavabyrd** [comment]
so far I think this looks good but should we hold off to get #25 added and then we can adjust from there? What do you think @pienkowb ? My thoughts are we make the change here, then move on upstream to graph-ts, modify the tendermint branches, firehose, etc?

File: .clang-format [hunk 2]
```diff
@@ -0,0 +1,7 @@
+BasedOnStyle: google
```
Comment (pienkowb) [inline .clang-format:1]: We could add `Language: Proto` here.

File: gogoproto/gogo.proto [hunk 3]
```diff
@@ -28,7 +28,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-syntax = "proto2";
+syntax = "proto3";
```
Comment (pienkowb) [inline gogoproto/gogo.proto:31]: Was it necessary to make this change? I think we could use the original files here.

**Review by pienkowb — COMMENTED** [review]
This looks really good. Great work, Seb! I hope the bindings generated based on these files will work, otherwise we need to flatten the structure (just like with the Tendermint types).

We still need to reconcile this with my PR but some of the changes we made are the same (e.g. moving 3rd-party protos to the top-level directory), so it shouldn't be a problem.

File: Makefile (hunk 1 above)
Comment (Seb-figment) [reply Makefile]: done

File: gogoproto/gogo.proto (hunk 3 above)
Comment (Seb-figment) [reply gogoproto/gogo.proto:31]: I don't remember changing this. Probably not needed.

Comment (pienkowb) [reply gogoproto/gogo.proto:31]: Alright, so let's use the original file.

File: .clang-format (hunk 2 above)
Comment (Seb-figment) [reply .clang-format:1]: done

**Pull request merged**